BUILD.md

```diff
@@ -0,0 +1,128 @@
+# **BUILD**
+
+## **Table of contents** 
+### 1. [**About**](#about)
+### 2. [**Environment**](#environment)
+### 3. [**Development**](#development)
+>#### 3.1. [**Install and Run**](#install-and-run)
+>#### 3.2. [**Style Guide**](#style-guide)
+>#### 3.3. [**Tests**](#security)
+>##### 3.3.1 [**E2E**](#e2e)
+>##### 3.3.2 [**Unitary Tests**](#unitary-tests)
+>#### 3.4 [**Security**](#security)       
+### 4. [**Production**](#production)
+
+
+## **About**
+The **BUILD.md** is a file to check the environment and build specifications of **horusec-cli** project.
+
+
+## **Environment**
+
+- **Golang**: ^1.17.X
+
+## **Development**
+
+Use a command-line interface (CLI) of your choice to run the following commands to download and install the dependencies.
+
+### **Install and Run**
+
+From the root of the project, run the command below to download the dependencies:
+
+```go
+go mod download
+```
+
+The command below allows the execution of the program, run it in the root folder:
+
+```bash
+make build-dev && ./horusec
+```
+
```

Comment:
I think that would be better split this section like this:
----

The command below build the Horusec cli for development

```bash
make build
```

After build successful you can execute running
```bash
./horusec
```

Or on Windows
```bash
horusec
``` 

With that we make it clear how to build and how to run the application